nest-forms-backend/src/nases/nases.service.ts

```diff
@@ -434,6 +453,42 @@ export default class NasesService {
       )
     }
 
+    if (form.formDataJson == null) {
+      throw this.throwerErrorGuard.UnprocessableEntityException(
+        FormsErrorsEnum.EMPTY_FORM_DATA,
+        FormsErrorsResponseEnum.EMPTY_FORM_DATA,
+      )
+    }
+
+    if (formDefinition.isSigned) {
+      if (!form.formSignature) {
+        throw this.throwerErrorGuard.UnprocessableEntityException(
+          NasesErrorsEnum.SIGNATURE_MISSING,
+          NasesErrorsResponseEnum.SIGNATURE_MISSING,
+        )
+      }
+
+      try {
+        verifyFormSignature(
+          formDefinition,
+          form.formDataJson,
+          form.formSignature,
+        )
+      } catch (error) {
+        if (error instanceof VerifyFormSignatureError) {
+          const { error: errorEnum, message: errorMessage } =
+            verifyFormSignatureErrorMapping[error.type]
+          throw this.throwerErrorGuard.UnprocessableEntityException(
+            // eslint-disable-next-line custom-rules/thrower-error-guard-enum
```

Comment:
NasesErrorsEnum should be valid here - add comment that this is a bug ?

Reply:
Yes, but it is mapped, and the ESLint rule won't work in this case.